app/src/androidTest/java/org/akvo/flow/ApplicationTest.java

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
This copyright header is outdated, Stellan has made some changes to it and it now looks like in this file:
https://github.com/akvo/akvo-flow-mobile/blob/develop/app/src/main/java/org/akvo/flow/service/ApkUpdateService.java